staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -81,7 +82,7 @@ type StartFunction func(ctx context.Context, hashValue uint64) (execute bool, af
 type RequestDigest struct {
 	RequestInfo *request.RequestInfo
 	User        user.Info
-	Width       uint
+	Width       *fcrequest.Width
```

Comment:
Sorry - I wasn't clear. I only meant changing parameters of functions.
I wouldn't change the fields in types (we should be passing those structs by pointers anyway).

So I would revert here and in `request` type.

Reply:
i reverted `RequestDigest` and `request` type